manifests/global_defs.pp

```diff
@@ -74,6 +77,7 @@
   $enable_snmp_rfcv2                              = undef,
   $enable_snmp_rfcv3                              = undef,
   $enable_traps                                   = undef,
+  Optional[Boolean] $enable_dbus                  = undef,
```

Comment:
How about this?
```suggestion
  Boolean $enable_dbus                            = false,
```

Reply:
I find your suggested code better, too.
I orientated myself on the surrounding code using `undef` all over.

Reply:
Mhm makes sense, can you update the docs as well?